src/clients/go/README.md

```diff
@@ -7,53 +7,133 @@ The TigerBeetle client for Go.
 Make sure to import `github.com/tigerbeetledb/tigerbeetle-go`, not
 this repo and subdirectory.
 
-For example:
+## Installation
 
 ```bash
-$ cat test.go
-package main
+printf 'package main
 
 import _ "github.com/tigerbeetledb/tigerbeetle-go"
 import "fmt"
 
 func main() {
   fmt.Println("Import ok!")
 }
-
-$ go mod init tigerbeetle-test
-$ go mod tidy
-$ go build
-$ ./tigerbeetle-test
-Import ok!
+' > test.go
+go mod init tbtest
+go mod tidy
+go build
```

Comment:
Using `` ```console `` and `$` would look nicer, but wouldn't be copy-paste friendly

Reply:
Yeah I am constantly going back and forth about this. I think we should stick with `$`. I'll make the change.

Reply:
done